lib/orm/associations/belongs-to.js

```diff
@@ -13,7 +15,13 @@ import assert from "../../assert";
  */
 export default class BelongsTo extends Association {
   get identifier() {
-    return `${camelize(this.key)}Id`;
+    if (typeof identifierCache[this.key] === "string")
```

Comment:
Any reason for the `typeof` check vs `if (identifierCache[this.key])`?

Reply:
The return type is always a string and doing a typeof check prevents theoretical false-positives, like `key: ""` which would never use the cache.

I can remove it as an empty string key is probably not likely to appear.

Reply:
Ok cool, great point. Let's keep the `typeof` then!